task_manager/routes/tasks.py

```diff
@@ -23,7 +23,8 @@ def model_post_create_task():
 def model_fetch_task(task_id):
     task = Task.query.all()
     headers = {'Content-Type': 'text/html'}
-    return make_response(render_template('index.html', tasks=task), 200, headers)
+    return make_response(render_template('index.html', tasks=task), 200,
```

Comment:
i dont see a need for this change

Reply:
line is too long for flake8

> i dont see a need for this change

the line is too long for flake8

Reply:
I might have to make the line lengths longer on flake-8, but this is fine